dashboard/src/components/AppView/DeploymentTable.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+
+import { IResource } from "shared/types";
+import DeploymentItem from "./DeploymentItem";
+import DeploymentTable from "./DeploymentTable";
+
+it("renders a deployment ready", () => {
```

Comment:
I think we can combine this with the one about two deployments, but up to you!

Reply:
I want to keep this test as simple as possible because it's comparing the result with a snapshot. The other test is less strict about the layout as far as it renders two elements.